frontend/src/components/yellowbutton/YellowButton.tsx

```diff
@@ -0,0 +1,12 @@
+import classes from './YellowButton.module.css';
+
+function YellowButton() {
+    return (
+        <div>
+          <button>
+            <p className={classes.text} >Next</p>
```

Comment:
we shouuuuld be able to just write this in the <button> itself and apply the text styles to said button